src/threechances.cpp

```diff
@@ -24,6 +24,59 @@ ThreeChances::~ThreeChances() {
 void ThreeChances::initialize(HWND hwnd) {
 	Game::initialize(hwnd); // throws GameError
 
+	// initialize map class
+	stage = new Stage;
+
+	// Legend
+	// 0 - border
+	// 1 - 1st floor path
+	// 2 - 2nd floor path
+	// 3 - stairs
+	// 4 - water
+	// 5 - obstacle
+	// 6 - blockage
+
+	std::vector<std::vector<int>> layout = {
```

Comment:
Create a new file named 'mapLayout.h' or something relevant and store there. The main game file shouldn't be used to store large data.